server/manager/src/main/java/org/apache/accumulo/manager/ManagerTime.java

```diff
@@ -85,20 +85,38 @@ public class ManagerTime {
    * start.</li>
    * </ul>
    */
-  private final AtomicReference<Duration> skewAmount;
+  private AtomicReference<Duration> skewAmount;
 
-  public ManagerTime(Manager manager, AccumuloConfiguration conf) throws IOException {
-    this.zk = manager.getContext().getZooSession().asReaderWriter();
-    this.manager = manager;
+  ManagerTime() {
+    this.manager = new AtomicReference<>(null);
+  }
+
+  // Once it's set call constructor
+  public void setManager(Manager manager) throws IOException {
```

Comment:
Its important this class is non functional before this method is called.  Would be good to add a unit test like the following for this class.  This will ensure that code that uses this before setManager is called does not get an incorrect time.  

```java
     var mt = new ManagerTime();
     assertThrows(???, ()->mt.getTime());
```

Seems like the current code may throw an NPE which is good.